entry.html

```diff
@@ -2822,14 +2856,14 @@ <h5 class="mb-3"><label for="signature-data">Signaturdaten</label></h5>
     function updateFingerPrint() {
         let fingerPrintElement = document.getElementById("finger-print");
         let entryForm = document.getElementById("entry-form");
-        let submitButton = document.getElementById("submit-button");
+        // let submitButton = document.getElementById("submit-button");  delegated to submitButtonToggler
```

Comment:
Don't leave commented code lying around, it is hard to say how old the comment is and, in isolation, it does not provide insight on the overall structure. We have git to see what changed when. And the code should be, in the best of all worlds ;-), always so clear and structured, that the concepts can be deduced